docs/content/Getting-Started/Overview.mdx

```diff
@@ -7,6 +7,8 @@ menuOrder: 1
 isDisableFeedbackBlock: false
 ---
 
+Cube is a cloud-native application, designed to run as a set of containers.
```

Comment:
```suggestion
Cube is a cloud-native application, packaged and distributed as Docker images and designed to run in a containerized environment.
```

WDYT?

Reply:
Could maybe add a link to https://docs.docker.com/get-started/overview/#images

Reply:
Sounds great!